step-capstone/src/components/TravelObjectForms/AddEvent.js

```diff
@@ -1,146 +1,165 @@
-import React, { useEffect } from 'react'
-import { useState } from 'react'
+import React, { useEffect } from "react";
+import { useState } from "react";
 import {
-    TextField,
-    Grid,
-    Checkbox,
-    FormControlLabel,
-    Box
-} from '@material-ui/core';
+  TextField,
+  Grid,
+  Checkbox,
+  FormControlLabel,
+  Box,
+} from "@material-ui/core";
 import {
-    MuiPickersUtilsProvider,
-    DateTimePicker
-} from '@material-ui/pickers';
+  MuiPickersUtilsProvider,
+  KeyboardDateTimePicker,
```

Comment:
the only change in add[TravelObject] files is replacing DateTimePicker with KeyBoardDateTimePicker. Please ignore the add[TravelObject] files.

Reply:
Sorry for the confusion.